recipes/eagle2/build.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#set -euo pipefail
+
+echo "Building Eagle2 version ${PKG_VERSION}"
+
+cd src
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)